lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj

```diff
@@ -41,6 +41,9 @@
 		<PackageReference Include="Xamarin.GooglePlayServices.Maps" Version="118.2.0.3" />
 	</ItemGroup>
 
+	<ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
```

Comment:
same. Dislike. And please open prs on your previous branches if u need the code that is not merged yet. it is hard to track the changes.

Reply:
Now I have the opposite comment, why did u do all the MapId stuff in the ClusteredMap handler if everything needed for it lives in the regular map sdk packages?

Reply:
What? I did exactly what you suggested and now you want me to undo it to the previous one? You should have analyzed the PR better, if you don't want to approve it as it is you can close this and #84.

It's boring trying to contribute with so much coming and going.

Reply:
No, i had a comment about the geojson since the code needed to do that lives in the utils package. MapId lives in the maps sdk packages, so it could be done not in the clustering project.

I appreciate your contribution but please be open to criticism as well

Reply:
I had done it the way you said was the most correct, it fits the same criteria you used for geojson, it is a utility although it does not need the utils library.

I undid the other PR with a very weak argument of lib dependency for something that is clearly part of the Google Maps core, although it uses another plugin, I will not change this one because it was correct, I adjusted it to be the same as the other one because it is obvious that this one fits the utils criteria.

Open to criticism? I undid the other PR with weak arguments, but this one was correct, if it was to be kept, I should have warned you, I will not redo it, if you do not want to approve it I will not contribute here anymore, this is boring.

End of the road so far.

Reply:
Maybe after you blow your steam and the anger towards me passes you will have the desire to finish heatmaps  :) Leaving that one open
(Should be done in the clustering project!)